### ISSUE TITLE
Create standard units for Measurement concepts

### ISSUE THREAD

**Issue by cgreich** [OHDSI/Vocabulary-v5.0#451]
https://forums.ohdsi.org/t/converting-lab-results-into-preferred-unit-at-etl-time/11856

**Comment by dimshitc** [comment]
When we convert some data, we need to store the original unit we converted from.

So to Patrick, we do need to add a column to the CDM.

why is it in the vocabulary repo?

**Comment by cgreich** [comment]
The issue about the missing field in MEASUREMENT is here: https://github.com/OHDSI/CommonDataModel/issues/259.
This is about create a link between Measurement Concepts and Unit Concepts "Is measured in" and "used to measure".

**Comment by Alexdavv** [comment]
> had a plan to work on LOINC.

Is LOINC gonna be alive once we implement [this](https://github.com/OHDSI/Vocabulary-v5.0/issues/452)?

**Comment by dimshitc** [comment]
Well, we can add units to whatever vocabulary is standard. For SNOMED it would be not so obvious as it doesn't specify the units nor even dimension. So these tasks needs to be done together.

**Comment by cgreich** [comment]
You mean, is it possible that SNOMED always wins and LOINC becomes the source concept?

**Comment by dimshitc** [comment]
I don't know yet who wins, I'm just saying that it's tricky:)

**Comment by Alexdavv** [comment]
> You mean, is it possible that SNOMED always wins and LOINC becomes the source concept?

Yes. Not always, but for those top-something cases that we're able to cover.

I'd always prefer LOINC, except we have an uncertain source. For those cases, the SNOMED is also needed. So in fact, they're not duplicates 😃 

How about this: we'll make those SNOMED classifications and will allow them to be used as the _concept_id?

**Comment by cgreich** [comment]
So, yes, let's work on this and the [deduping one](https://github.com/OHDSI/Vocabulary-v5.0/issues/452) at the same time.

**Comment by vojtechhuser** [comment]
I asked repeatedly LOINC leaders if they are willing to add it to LOINC (change example units) into CANONICAL units and the reply was:  (pasted below)

> As we discussed before, LOINC cannot include canonical UCUM units because the LOINC Property is one of the defining aspects of a given LOINC term, not the unit of measure, and each Property corresponds to multiple UCUM units. We can publish AoU and N3C UCUM lists on the UCUM website, but we can’t add them to the LOINC table as anything other than example units.

**Comment by Alexdavv** [comment]
Right, but the examples provided are highly relevant and the only thing we have to do is pick one among others.

**Comment by AEW0330** [comment]
It would be good to include Richard Moffit @rmoffitt from N3C in this conversation to learn what he's done standardize units in the OMOP instance there.

**Comment by rmoffitt** [comment]
We have concept sets for about 50 measurements, each with a selected unit. Then formulae to convert data from the other units to that one canonical unit. We store that in a new column, harmonized_value_as_number.

**Comment by Alexdavv** [comment]
> We have concept sets for about 50 measurements, each with a selected unit. Then formulae to convert data from the other units to that one canonical unit. We store that in a new column, harmonized_value_as_number.

How did you select one from others?

**Comment by rmoffitt** [comment]
We have many physicians on the team, so we relied on their experience when prioritizing which labs to harmonize as well as which canonical units to aim for. They were also central in our concept set review process.

**Comment by Alexdavv** [comment]
> We have many physicians on the team, so we relied on their experience when prioritizing which labs to harmonize as well as which canonical units to aim for. They were also central in our concept set review process.

Great! Can you share it so that we can start thinking about making it a part of OMOP vocabulary?
